src/osd/OSD.cc

```diff
@@ -7756,6 +7756,8 @@ MPGStats* OSD::collect_pg_stats()
 	min_last_epoch_clean_pgs.push_back(pg->pg_id.pgid);
       });
   }
+  pool_set.emplace(-1); // add "meta" pool explicitly as
```

Comment:
I would love to have a `CONSTANT` defined for this magic.

Reply:
Done